src/components/WonderCart.js

```diff
@@ -54,14 +58,18 @@ export function WonderCart(props) {
               </span>
             </div>
             <div className="nfd-flex-none">
-              <Button
-                type="button"
-                variant="upsell"
-                isLoading={showInProgress}
-                onClick={installWonderCart}
-              >
-                {__("Install now", "wp-module-ecommerce")}
-              </Button>
+                <Button
+                  type="button"
+                  as={canAccessGlobalCTB ? "a" : "button"}
+                  data-ctb-id={canAccessGlobalCTB ? "f95ccf1e-3028-4ea7-b2c2-847969348e8b" : null}
+                  data-action={canAccessGlobalCTB ? "load-nfd-ctb" : null}
+                  href={canAccessGlobalCTB && "https://my.bluehost.com/hosting/app?utm_source=wp-marketplace&utm_medium=brand-plugin&utm_campaign=wordpress-ad&utm_content=buynow#/marketplace/product"}
```

Comment:
We should configure the destination URL in the brand plugin, not in the module. We should be setting the URL in the container in the brand plugin, then fetching it from the container in the module. This way things can work properly for our other brands as well.